tools/twix/src/panels/enum_plot.rs

```diff
@@ -97,17 +97,31 @@ pub struct EnumPlotPanel {
 impl Panel for EnumPlotPanel {
     const NAME: &'static str = "Enum Plot";
 
-    fn new(nao: Arc<Nao>, _value: Option<&serde_json::Value>) -> Self {
-        Self {
+    fn new(nao: Arc<Nao>, value: Option<&Value>) -> Self {
+        let output_key = value
+            .and_then(|value| value.get("subscribe_key"))
+            .and_then(|value| value.as_str())
+            .map(String::from)
+            .unwrap_or_default();
+
+        let mut panel = Self {
             nao,
             changes: Vec::new(),
             messages_count: 0,
             change_buffer: None,
             scroll_position: 0.0,
             viewport_width: 1.0,
-            output_key: String::new(),
+            output_key,
             viewport_mode: ViewportMode::Full,
-        }
+        };
+        panel.subscribe();
+        panel
+    }
+
+    fn save(&self) -> Value {
+        json!({
+            "subscribe_key": self.output_key.clone()
```

Comment:
Remove unnecessary clone
```suggestion
            "subscribe_key": self.output_key
```